src/index.js

```diff
@@ -1,6 +1,53 @@
 import Display from '../module/Display.js';
 import './style.css';
+import Tasks from '../module/Task.js';
+import LocalStorage from '../module/LocalStorage.js';
 
-document.addEventListener('DOMContentLoaded', () => {
-  Display.addList();
-});
\ No newline at end of file
+const form = document.querySelector('.forms');
+const listContainer = document.querySelector('.showList');
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const des = document.getElementById('desc').value;
+  const index = LocalStorage.idGenerator();
+  const id = LocalStorage.idGenerator();
+  const task = new Tasks(des, index, id);
+  Display.addList(task, id);
+  LocalStorage.addLocal(task);
+  document.getElementById('desc').value = '';
+});
+
+listContainer.addEventListener('click', (e) => {
+  const eventClass = e.target.className;
+
+  if (eventClass === 'mycheckbox') {
+    e.target.nextSibling.classList.toggle('active');
+    Display.changebool(e.target.id);
+  } else if (eventClass === 'displayP') {
+    const list = LocalStorage.getlist();
+    const eventinput = e.target.previousSibling.id;
+
+    e.target.addEventListener('change', (e) => {
+      list[eventinput - 1].des = e.target.value;
+      localStorage.setItem('list', JSON.stringify(list));
+    });
+  } else {
+    const removebtn = e.target.parentNode.previousSibling.firstChild.id;
+    Display.renameLocal(removebtn, e.target);
+  }
+});
```

Comment:
- [ ] Your tasks list is not keeping the localStorage state after a page refresh. Completed tasks are not checked anymore meanwhile their status is `true`. You should probably check your getListt function to fix that.

![onchange](https://user-images.githubusercontent.com/40503704/184195519-0c5a941c-5a4f-47e9-bedf-c2beca2f9dc5.PNG)